third_party/silabs/efr32_sdk.gni

```diff
@@ -33,7 +33,10 @@ declare_args() {
 
   # Build openthread with prebuilt silabs lib
   use_silabs_thread_lib = false
-  enable_openthread_cli = true
+
+  # This undefined value (-1) is used to set a conditional default
+  # when users don't set it in build args
+  enable_openthread_cli = -1
```

Comment:
this is ugly. Make it a string and use "default", "enabled", "disabled" (or auto or similar).

Ideally you would have the default in "declare_args" however then you need to reference use_rs9116 and such (not have them computed).

Reply:
Please use "auto", "enabled", "disabled" as possible options in this case. I am not a fan of a tri-bool with values "-1", "true", "false" as that seems tricky and tricky code is hard to maintain.

Reply:
Scratch that, The initial issue I had with using your suggestion was probably done by the one typing ( me :( )

conditional default value work in the declare args with those other args